dist/api_data.py

```diff
@@ -1319,11 +1319,15 @@ def __ge__(self, other):
         Display the contents of in-memory pages as they are verified,
         using the application's message handler, intended for debugging''',
         type='boolean'),
+    Config('dump_timestamps', 'false', r'''
```

Comment:
Would it be better to call it dump_timepairs?

Reply:
I will go with whatever we call (timestamp + txn-id) in durable history branch.

Reply:
Yeah timepair is better.